### DISCUSSION TITLE
Syncing up Prism.Core with Prism.Avalonia

### DISCUSSION THREAD

**Discussion by DamianSuess** [PrismLibrary/Prism#2874] [Show and Tell] [answered]
Hi @dansiegel and @brianlagunas,

May I start with, thank you both for maintaining this amazing project all these years! Seriously, thank you! As the maintainer of [Prism.Avalonia](https://github.com/AvaloniaCommunity/Prism.Avalonia), it's a lot of fun helping the Avalonia community as well as using this library for my own Linux/Mac and MAUI projects.

For other readers who don't know, the project shares similarities with WPF/MAUI and even implements the IDialogService (_love it_).

## Question Time!
With Prism v9.x coming around the corner soon, I was wondering if you wouldn't mind sharing what else you had in store for the project's upcoming roadmap. Overall, I'd like to try to be proactive as possible to stay aligned with the Prism.Core library.

Also, I'm working on a Prism.Avalonia.Template pack for DotNet CLI and Visual Studio. Would it be okay if I piggyback the Prism.Avalonia "Add New Item" templates in Visual Studio under the same folder structure as Prism's? I'm open for suggestions

![image](https://github.com/PrismLibrary/Prism/assets/1216684/c35a3e76-1fcc-44f8-8998-446089e4bcb3)

**Comment by brianlagunas** [comment, marked as answer]
The most important thing you should be aware of regarding Prism.Core is that we have pulled the event aggregator out into its own nuget package. Prism.Core will take a dependency on the new Prism.Events package to maintain backwards compatibility.

As far as a roadmap goes, we don't have one. Our work will focus on areas that are needed by the maintainers and the community. This may include focus on Uno Platform, MAUI, and some tooling updates.

Feel free to piggyback on the existing templated groups.

**Reply by DamianSuess** [reply]
Thank you, Brian, for getting back so quickly.

I appreciate the rundown and the thumbs-up on the project templates. Frankly, I was surprised to EventAggregator diverged in #2867, which kind of sparked me to reach out to you guys. I assumed it was to make things more lightweight and optional.

Best wishes to you guys and your families.  If you ever find yourselves coming through Pittsburgh, a round is on me. 🍻
